ephios/templates/registration/login.html

```diff
@@ -9,16 +9,27 @@
 
 {% block content %}
     {% translate "Login" as login_message %}
+    {% oidc_clients as clients %}
```

Comment:
can we build this as template var instead of a tag?

Reply:
difficult, because that is the built-in django login view and not one of our own

Reply:
I see